src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs

```diff
@@ -20,7 +20,7 @@ public sealed class RankerMetrics
         /// <a href="https://en.wikipedia.org/wiki/Discounted_cumulative_gain">Discounted Cumulative gain</a>
         /// is the sum of the gains, for all the instances i, normalized by the natural logarithm of the instance + 1.
         /// Note that unline the Wikipedia article, ML.Net uses the natural logarithm.
-        /// <a href="https://github.com/dotnet/machinelearning/tree/master/docs/images/DCG.png"></a>
+        /// <image href="https://github.com/dotnet/machinelearning/tree/master/docs/images/DCG.png"></image>
```

Comment:
is 'href' appropriate now? Or should it be 'src' ?

Reply:
It should be src.

Reply:
I'd updated with 'src'